vscode-lean4/src/utils/clientProvider.ts

```diff
@@ -170,14 +169,22 @@ export class LeanClientProvider implements Disposable {
         return Array.from(this.clients.values());
     }
 
-    getClientForFolder(folder: Uri) : LeanClient | undefined {
+    getKeyForFolder(folder: Uri | null) : string{
```

Comment:
What is a key for a folder? Please put some explanation in a comment.

Reply:
fixed